src/uct/api/uct.h

```diff
@@ -265,6 +265,26 @@ enum uct_iface_event_types {
 
 /**
  * @ingroup UCT_RESOURCE
+ * @brief  Flush modifiers.
+ */
+enum uct_flush_flags {
+    UCT_FLUSH_FLAG_LOCAL    = 0,            /**< Guarantees that the data
+                                                 transfer is completed but the
+                                                 target buffer may not be
+                                                 updated yet.*/
+    UCT_FLUSH_FLAG_CANCEL   = UCS_BIT(0)    /**< The library will do the best
+                                                 effort to cancel all
+                                                 incompleted operations. User
+                                                 should be aware about the fact
+                                                 there is a chance that some
+                                                 operation will not be completed
+                                                 and been ready to handle all
+                                                 the relevant callbacks. */
```

Comment:
Suggested rewording for grammar and clarity (assuming I've gotten the semantics right):
The library will make a best effort attempt to cancel all uncompleted operations.  However, there is a chance that some operations will not be canceled in which case the user will need to handle their completions through the relevant callbacks.